--- a/config/auth.php
+++ b/config/auth.php
@@ -67,25 +67,25 @@
     'providers' => [
 
     	// Comment out or remove below for LDAP
-        'users' => [
-            'driver' => 'eloquent',
-            'model'  => App\User::class,
-        ],
-
-        // Uncomment below for LDAP
         // 'users' => [
-        // 	'driver' => 'ldap',
-        // 	'model' => LdapRecord\Models\ActiveDirectory\User::class,
-        // 	'rules' => [],
-        // 	'database' => [
-        // 		'model' => App\User::class,
-        // 		'sync_passwords' => false,
-        // 		'sync_attributes' => [
-        // 			'name' => 'cn',
-        // 			'email' => 'mail',
-        // 		],
-        // 	],
+        //     'driver' => 'eloquent',
+        //     'model'  => App\User::class,
         // ],
+
+        // Uncomment below for LDAP
+        'users' => [
+        	'driver' => 'ldap',
+        	'model' => LdapRecord\Models\ActiveDirectory\User::class,
+        	'rules' => [],
+        	'database' => [
+        		'model' => App\User::class,
+        		'sync_passwords' => false,
+        		'sync_attributes' => [
+        			'name' => 'uid',
+        			'email' => 'mail',
+        		],
+        	],
+        ],
     ],
 
     /*
